### PR TITLE
Chore: refactor icons

### PR DESCRIPTION
**Pull request by wingfeatherwave**
Big icon refactor. 

* no more deprecated icons
* use only icons that are in figma or from the [Feather library ](https://feathericons.com/)
* cleanup directory structure

Lots of files added / changed in this PR.  Please check the vercel link for visual testing.

### CODE REVIEW

File: src/components/FormModal.tsx [hunk 1]
```diff
@@ -149,7 +146,6 @@ export const FormModal = ({
             }}
             disabled={submitDisabled}
           >
-            {icon || (!submitText && <DeprecatedSaveIcon />)}
```
Comment (CryptoGraffe) [inline src/components/FormModal.tsx:152]: This intentional?

File: src/components/OrgLogoUpload.tsx [hunk 2]
```diff
@@ -104,8 +105,8 @@ export const OrgLogoUpload = ({
                 background: '$white',
                 boxShadow: '$heavy',
                 borderRadius: '$3',
-                paddingLeft: '$2xl',
-                height: '$2xl',
+                paddingLeft: '64px',
```
Comment (CryptoGraffe) [inline src/components/OrgLogoUpload.tsx:108]: whats the story here

File: src/icons/__generated/CoinbaseColor.tsx [hunk 3]
```diff
@@ -6,26 +6,26 @@
...
 import { SvgIcon, SvgIconProps } from '../../ui/SvgIcon/SvgIcon';
 const css = {
-  '& path': {
-    stroke: 'none',
-  },
+  // '& path': { stroke: 'none' }
+  // '& path': { stroke: 'none' }
 };
-export const SvgHamburger = (props: SvgIconProps) => (
+export const SvgCoinbaseColor = (props: SvgIconProps) => (
```
Comment (CryptoGraffe) [inline src/icons/__generated/CoinbaseColor.tsx:12]: These rename diffs are bizarre, the file is renamed to something totally diff??

File: src/ui/index.ts [hunk 4]
```diff
@@ -23,6 +23,3 @@ export * from './TextArea/TextArea';
 export * from './TextField/TextField';
 export * from './Tooltip/Tooltip';
 export * from './Icon/Icon';
-
-// icons
-export * from './icons';
```
Comment (CryptoGraffe) [inline src/ui/index.ts:28]: whats the deal here? just curious

File: src/icons/__generated/CoinbaseColor.tsx (hunk 3 above)
Comment (wingfeatherwave) [reply src/icons/__generated/CoinbaseColor.tsx:12]: that is really bizarre.  why would this happen?

File: src/ui/index.ts (hunk 4 above)
Comment (wingfeatherwave) [reply src/ui/index.ts:28]: I was thinking that it was no longer needed.  Perhaps I'm wrong?

File: src/components/OrgLogoUpload.tsx (hunk 2 above)
Comment (wingfeatherwave) [reply src/components/OrgLogoUpload.tsx]: This was for the org avatar upload.  I can do it with a calc, so I'll make that change.  

Screen Shot 2022-09-16 at 3 55 56 PM

File: src/components/FormModal.tsx (hunk 1 above)
Comment (wingfeatherwave) [reply src/components/FormModal.tsx:152]: Yes, we don't need a save icon in the primary submit button that says "save", IMO

Comment (wingfeatherwave) [reply src/components/FormModal.tsx:152]: nvm... I can think of a case for keeping it.  I'll revert.

File: src/ui/index.ts (hunk 4 above)
Comment (wingfeatherwave) [reply src/ui/index.ts:28]: .... ah, that was a relic of when there were some icons stored in /ui.  yea, no longer need to import those.  
Screen Shot 2022-09-16 at 4 20 50 PM

**Review by CryptoGraffe — APPROVED** [review]

**Pull request merged**